doc/Requirements.tex

```diff
@@ -113,22 +105,11 @@ \section{Runtime Library}
 
 What distinguishes runtime library material from active CubedOS modules is that none of the
 runtime library components contain tasks or perform blocking operations (such as delays).
-Notice, however, that some library components may wish to send messages to other active
```

Comment:
Library components now can't send messages because they don't have a mailbox to send it from. I guess you could pass in your mailbox, but this whole business of libraries sending messages seems bizzare and definitely doesnt belong in the "Requirements" section.